### PR TITLE
Throwing Knives: Syndicate Kit

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

This PR adds throwing knives and a syndicate bundle in the bundle category for 6 TC which comes with 4 throwing knives in a box. Each knife does 10 slash + 15 pierce damage on throw, and 5 flash per hit (10 DPS). This makes it better than a kitchen knife, but worse than a combat knife, which is what security is fitted with.

## Why / Balance

This kit is built for weakening an enemy from a distance (from chasing, etc) while being able to fight back if cornered into a 1v1 CQC fight. Sure, you can kill several people with the kit; hence why it's so expensive, but there are drawbacks. Firstly, once it embeds into a target they are able to pull it out. Secondly, when you are pulling them out of a target, YOU are vulnerable. Thirdly, if you miss your hits the target will probably not die. Good things you have four knives and can stab them to death.. better than punches!

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**It embeds and kills in 4 throws on an unarmoured target.**
image

**Item description.**
image

**4 knives.**
image

**Kit on uplink.**
[IMG NEEDS UPDATING]

**Changelog**

:cl: Ubaser
- add: You can now purchase a set of 4 throwing knives in the uplink as a bundle for 12 TC.

### CODE REVIEW

**Comment by UbaserB** [comment]
> Definitely no problem with _25_ damage knives with spamming quick store and throw key
> 
> Somehow only does 3 melee :trollface:

it's 5 damage and has 2x speed which actually feels pretty powerful ingame.

**Comment by K-Dynamic** [comment]
image

ok you get your wish

- - -

12 tc pack of four 25 dmg knives is expensive and hard to aim for most people, will probably be exploited by keybind users to instacrit silently

**Comment by UbaserB** [comment]
Now 8 tc and 15 pierce + 10 slash

**Comment by TurboTrackerss14** [comment]
>yet will be exploited by keybind users to instacrit silently

Except that throwing has a "repeat" limitation now, locking out throws and such for a short time to stop said macro/keybind abusers. 
All round fun(ny) item. can't wait to roleplay as a knife juggling clown "accidentally" throwing my knives at the crowd

**Comment by UbaserB** [comment]
> > Now 8 tc and 15 pierce + 10 slash
> 
> 8 tc still feels like a lot, I'd price it at 6 personally. That's 2 TC per knife, for 2 tc you can get a lot of much more effective murder tools.

isnt an edagger like 3 or 4, this is not much worse and definitely a lot more stealthy.

**Comment by Dutch-VanDerLinde** [comment]
> > > Now 8 tc and 15 pierce + 10 slash
> 
> > 
> 
> > 8 tc still feels like a lot, I'd price it at 6 personally. That's 2 TC per knife, for 2 tc you can get a lot of much more effective murder tools.
> 
> 
> 
> isnt an edagger like 3 or 4, this is not much worse and definitely a lot more stealthy.

The e-dagger is 2 TC

**Comment by Dutch-VanDerLinde** [comment]
Maybe 6 TC?

**Comment by asperger-sind** [comment]
> Maybe 6 TC?

Yeah, 4-6TC would make this a pretty solid option imo.

File: Resources/Textures/Objects/Storage/boxes.rsi/meta.json [hunk 1]
```diff
@@ -152,12 +155,12 @@
             "name": "writing_of_doom"
         },
         {
-			"name": "headset"
-		},
+	    "name": "headset"
+        },
```
Comment (metalgearsloth) [inline Resources/Textures/Objects/Storage/boxes.rsi/meta.json:159]:
```suggestion
        "name": "headset"
        },
```

File: Resources/Textures/Objects/Storage/boxes.rsi/meta.json [hunk 2]
```diff
@@ -152,12 +155,12 @@
...
         {
-			"name": "encryptokey"
-		},
-		{
+            "name": "encryptokey"
+	},
+	{
```
Comment (metalgearsloth) [inline Resources/Textures/Objects/Storage/boxes.rsi/meta.json:163]:
```suggestion
    },
    {
```

File: Resources/Textures/Objects/Storage/boxes.rsi/meta.json [hunk 3]
```diff
@@ -35,7 +35,7 @@
             "name": "sechud"
         },
         {
-			"name": "bottle"
+	    "name": "bottle"
```
Comment (metalgearsloth) [inline Resources/Textures/Objects/Storage/boxes.rsi/meta.json:38]:
```suggestion
        "name": "bottle"
```

**Pull request merged**